src/stores/FlairStore.js

```diff
@@ -48,6 +48,9 @@ class FlairStore extends EventEmitter {
             //      reject: () => {}
             //  }
         };
+        this._usersInFlight = {
+            // see _usersPending
```

Comment:
might be worth spelling out that it's literally the same schema as _usersPending - not just a "to understand this, please look at _usersPending"

Reply:
sure!